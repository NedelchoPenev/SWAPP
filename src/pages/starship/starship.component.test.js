import React from 'react';
import { shallow } from 'enzyme';

import StarshipPage from './starship.component';

describe('StarshipPage component', () => {

  it('should render StarshipPage component', () => {
    const mockProps = {
      starship: {
        name: 'Test name',
        model: 'Test model'
      }
      
    }
    expect(shallow(<StarshipPage {...mockProps}/>)).toMatchSnapshot();
  });
});