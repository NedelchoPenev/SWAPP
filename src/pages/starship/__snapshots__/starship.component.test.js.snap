// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StarshipPage component should render StarshipPage component 1`] = `
<styled.div>
  <h1>
    Test name
  </h1>
  <h2>
    (
    Test model
    )
  </h2>
  <styled.div>
    <StarshipDetails
      starship={
        Object {
          "model": "Test model",
          "name": "Test name",
        }
      }
    />
    <Radar />
  </styled.div>
</styled.div>
`;
